src/log.js

```diff
@@ -447,70 +449,6 @@ class MultiLogger {
   }
 }
 
-/**
```

Comment:
- why was this removed?
- why is this change together with the changes for bunyan?